spec/facility_factory_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'spec_helper'
+require './lib/vehicle'
+require './lib/vehicle_factory'
+require './lib/dmv_data_service'
+require './lib/facility_factory'
+
+RSpec.describe FacilityFactory do
+
+    let(:oregon) do
+        {:title=>"Albany DMV Office",
+            :zip_code=>"97321",
+            :website=>"http://www.oregon.gov/ODOT/DMV/pages/offices/albany.aspx",
+            :type=>"DMV Location",
+            :phone_number=>"541-967-2014",
+            :agency=>"Transportation, Department of ",
+            :location_1=>
+            {:latitude=>"44.632897",
+                :longitude=>"-123.077928",
+                :human_address=>"{\"address\": \"2242 Santiam Hwy SE\", \"city\": \"Albany\", \"state\": \"OR\", \"zip\": \"97321\"}"}
+            }
+    end
+
+    it "exists" do
+        facility_factory = FacilityFactory.new
+        expect(facility_factory).to be_an_instance_of(FacilityFactory)
+    end
+
+    it "can make a oregon office" do
```

Comment:
For this step, we would've liked to see you create a method that can parse ALL the Oregon output (i.e. an array of hashes that you turn into an array of facilities)